app/views/group_offers/index.html.slim

```diff
@@ -1,6 +1,88 @@
 h1= t_title
 
-= render 'group_offers_block', group_offers: @group_offers
+nav.navbar.section-navigation#filters
+  hr
+  ul.list-unstyled
+    li.li-search-form
+      = search_form_for @q do |f|
+        = f.search_field :search_volunteer_cont, class: 'search-field', data: { autocomplete: search_group_offers_path }, autofocus: true
+        = f.submit 'Suchen', class: 'search-submit'
```

Comment:
Use translations

Reply:
We have stopped using translations, except for tests (if needed). The application is going to be used only in german, so we decided to stop translating.

Reply:
ok